src/expensesAPI/expenses-controller.js

```diff
@@ -0,0 +1,175 @@
+'use strict';
+
+// Imports:
+const fs = require('fs');
+const path = require('path');
+const { getUsersFromDB } = require('../usersAPI/users-controller');
+
+// Path:
+const expensesDataBasePath = path.join(
+  __dirname, '..', 'data', 'expenses.json'
+);
+
+// - Array for tests only; expenses.json file
+// |-- could store all the data permanently.
+let expensesArr = [];
+
+// Function to get the data from the array/json/db:
+const getExpensesFromDB = () => {
+  const data = fs.readFileSync(expensesDataBasePath).toString();
+
+  // TODO:
+  // Return this from this func but NOT expensesArr to use .json as DB.
+  JSON.parse(data);
+
+  return expensesArr;
+};
+
+// Function to write the data into the array/json/db:
+const writeExpensesToDB = (newData) => {
+  fs.writeFileSync(expensesDataBasePath, JSON.stringify(newData, null, 2));
+
+  // TODO:
+  // Remove this line to use .json as DB.
+  expensesArr = newData;
+};
```

Comment:
helper / additional functions, or just some main functionality that you'll use for working with data should be described in separate service

create separate service both of users and expenses, and call needed functions from server inside the controllers